src/Indicator.vala

```diff
@@ -330,6 +335,49 @@ public class Session.Indicator : Wingpanel.Indicator {
         current_dialog.set_transient_for (indicator_icon.get_toplevel () as Gtk.Window);
         current_dialog.show_all ();
     }
+
+    private async void update_tooltip () {
+        string description;
+
+        if (server_type == Wingpanel.IndicatorManager.ServerType.SESSION) {
+
+            if (active_user == null) {
+                debug ("active_user is null. Getting active user");
+                active_user = yield manager.get_active_user ();
+            }
+
+            string active_real_name;
+            int n_online_users = (yield manager.get_n_active_and_online_users ()) - 1;
+
+            // If active user is still null, fallback to "Unknown"
+            if (active_user == null) {
+                critical ("Active user null. Cannot get real name");
```

Comment:
I'm not sure this should be critical because this is just a fallback and not a critical thing like segmentation fault, etc.

Reply:
Yeah it definitely doesn't sound like it should be critical. I'll change it up.